src/opera/test/pge/dswx_ni/test_dswx_ni_pge.py

```diff
@@ -490,7 +491,7 @@ def test_dswx_ni_pge_validate_output_product_filenames(self):
     def test_iso_metadata_creation(self):
         """
         Mock ISO metadata is created when the PGE post processor runs.
-        Successful creation of metadata is verified in test_dswx_s1_pge_execution().
+        Successful creation of metadata is verified in test_dswx_ni_pge_execution().
```

Comment:
Hmmmm.  dswx_s1 and dswx_ni are closely coupled through inheritance.  I believe the intent of the comment is to actually state that the verification of metadata creation action happens in the other pge (i.e. dswx_s1), and because this is shared code it does not have to be redone here.

Reply:
Could it be possible that this wouldn't be the case in future releases?

Reply:
@JimHofman I think in this case it's truly another cut and paste error. Our unit tests suites do not have any kind of inheritance relationship to them, even though the PGEs themselves might, so we still want a DSWx-NI specific version of this test.